scripts/make.py

```diff
@@ -160,7 +160,7 @@ def build_native_tools(self) -> None:
         dist_comps += self.llvm_binutils
 
         cmake_defs = {
-            'LLVM_TARGETS_TO_BUILD:STRING': 'ARM',
+            'LLVM_TARGETS_TO_BUILD:STRING': 'ARM;AArch64',
             'LLVM_DEFAULT_TARGET_TRIPLE:STRING': cfg.default_target,
```

Comment:
The `cfg.default_target is None` case needs to be handled here too (like in `build_clang`). Also, please consider factoring out common logic related to `cmake_defs` from `build_native_tools` and `build_clang`.

Reply:
Thanks, good catch.

Reply:
This has been fixed/implemented (including the factoring out common logic) in the updated patch series.